library/options.inc.php

```diff
@@ -1342,6 +1342,20 @@ function generate_form_field($frow, $currvalue)
         echo "<img class='signature' id='form_{$field_id_esc}_img' title='$description'
             data-pid='$cpid' data-user='$cuser' data-type='$datatype'
             data-action='fetch_signature' alt='Get Signature' src='" . attr($currvalue) . "'>\n";
+    } elseif ($data_type == 43) {
+        echo generate_select_list(
+            "form_$field_id",
+            $list_id,
+            $currvalue,
+            $description,
+            $showEmpty ? $empty_title : '',
+            "select-dropdown",
```

Comment:
also need to incorporate `$smallform` in above (like is done in data_type 1)

Reply:
```suggestion
            $smallform . " select-dropdown",
```

This will work?

Reply:
actually, it would be easier to maintain if incorporated this datatype into the `if ($data_type == 1 || $data_type == 33) {` above and then change this line to:
```
(($data_type == 43) ? $smallform . " select-dropdown" : $smallform),
```